components/benefit_sponsors/app/models/benefit_sponsors/model_events/benefit_application.rb

```diff
@@ -75,9 +75,7 @@ def notify_on_save
             end
           end
 
-          if is_transition_matching?(to: :canceled, from: [:enrollment_eligible, :active], event: :cancel)
-            is_benefit_coverage_renewal_carrier_dropped = true
-          end
+          _is_benefit_coverage_renewal_carrier_dropped = true if is_transition_matching?(to: [:canceled, :retroactive_canceled], from: [:enrollment_eligible, :active], event: :cancel)
```

Comment:
you've to remove `underscore` here; we're calling it [here](https://github.com/health-connector/enroll/blob/93cca3092df8b60bd56c873395471c32c031bea9/components/benefit_sponsors/app/models/benefit_sponsors/model_events/benefit_application.rb#L115)
if rubocop complains you either have to disable & re-enable it or use `underscore` while calling these.